src/panels/experimental-ui/hui-entities-vertical-card.js

```
@@ -0,0 +1,118 @@
+            compensating this with reduced padding */
+          line-height: 40px;
+          color: var(--primary-text-color);
+          padding: 4px 0 12px;
+        }
+        .header .name {
+          @apply --paper-font-common-nowrap;
+        }
+        .entities {
+          padding: 4px 0;
+          display: flex;
+          flex-wrap: wrap;
+          flex: 1 0 minmax(100px, 20%);
+          justify-content: space-around;
+        }
+        .entity {
+          padding: 0 4px;
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          cursor: pointer;
+        }
+      </style>
+
+      <ha-card>
+        <div class="header">
+          <div class="name">[[_computeTitle(config)]]</div>
+        </div>
+        <div class="entities">
+          <template is="dom-repeat" items="[[entities]]">
+            <template is="dom-if" if="[[_showEntity(item, hass.states)]]">
+              <div class="entity" on-click="_openDialog">
+                <div>[[_computeName(item, hass.states)]]</div>
+                <state-badge state-obj="[[_computeStateObj(item, hass.states)]]"></state-badge>
+                <div>[[_computeState(item, hass.states)]]</div>
+              </div>
+            </template>
+          </template>
+        </div>
+      </ha-card>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+      config: Object,
+      entities: {
+        type: Array,
+        computed: '_computeEntities(config)'
+      }
+    };
+  }
+
+  getCardSize() {
+    return 3;
+  }
+
+  _computeTitle(config) {
+    return config.title;
+  }
+
+  _computeEntities(config) {
+    return config && config.entities && Array.isArray(config.entities) ?
+      config.entities.filter(entity => VALID_DOMAINS.includes(computeDomain(entity))) : [];
+  }
+
+  _showEntity(item, states) {
+    return states && states[item] ? true : false;
```

I think we should currently lean to fail visible instead of fail silently

---

This is only to check if the entity exists in states, but yes, for the things we should show errors